src/Authentication/Services/SystemUserService.cs

```diff
@@ -132,5 +137,30 @@ public async Task<int> UpdateSystemUserById(SystemUserUpdateDto request)
         {
             return await _repository.CheckIfPartyHasIntegration(clientId, systemProviderOrgNo, systemUserOwnerOrgNo, cancellationToken);
         }
+
+        /// <inheritdoc/>
+        public async Task<Result<Page<SystemUser, string>>> GetAllSystemUsersByVendorSystem(
+            OrganisationNumber vendorOrgNo, 
+            string systemId, 
+            Page<string>.Request continueRequest, 
+            CancellationToken cancellationToken)
+        {
+            RegisterSystemResponse? system = await systemRegisterRepository.GetRegisteredSystemById(systemId);
+            if (system is null)
+            {
+                return Problem.SystemIdNotFound;
+            }
+
+            // Verify that the orgno from the logged on token owns this system
+            if (OrganisationNumber.CreateFromStringOrgNo(system.SystemVendorOrgNumber) != vendorOrgNo)
+            {
+                return Problem.SystemIdNotFound;
+            }
+
+            List<SystemUser>? theList = await _repository.GetAllSystemUsersByVendorSystem(systemId, cancellationToken);
+            theList ??= [];
+
+            return Page.Create(theList, 3, static theList => theList.Id);
```

Comment:
Pagesize på 3?

Reply:
det er for testing :-) skal fjerne magic number, og legge inn i appsettings eller annen config, så det ikke blir et hardkodet tall. Tenkte at 200 er et "riktig" tall for når vi setter det til tt02